### PR TITLE
docs(*): updates/changelog for v2.13.0 release

### PR DESCRIPTION
**Pull request by mboersma**
I also reversed the sorting on the changelogs index, since most-recent-first seems more user-friendly.

TODO:
- [x] update verification SHA in src/installing-workflow/chart-provenance.md
- [x] update verification SHA in src/roadmap/releases.md

### CODE REVIEW

**Comment by bacongobbler** [comment]
Everything else looks good here, just the verification SHAs left.

**Comment by mboersma** [comment]
Added the verification SHAs.

File: src/installing-workflow/chart-provenance.md [hunk 1]
```diff
@@ -34,31 +34,31 @@ Charts signed with this key can then be verified when fetched:
...
+Error: Failed to fetch provenance "https://charts.deis.com/workflow/workflow-v2.13.0.tgz.prov"
 ```
 
 Alternatively, the chart can also be verified at install time:
 
 ```
 $ helm install --verify deis/workflow --namespace deis
 NAME:   exiled-mink
-LAST DEPLOYED: Wed Mar  8 08:22:16 2017
+LAST DEPLOYED: Wed Apr  6 08:22:16 2017
```
Comment (vdice) [inline src/installing-workflow/chart-provenance.md]: Weird -- Today is actuall Wednesday April 5 -- was this hand-edited?  (typo then)

**Comment by vdice** [comment]
Other than typo, LGTM!

File: src/installing-workflow/chart-provenance.md (hunk 1 above)
Comment (mboersma) [inline src/installing-workflow/chart-provenance.md]: D'oh yeah--I updated the date by hand, but not the day. Fixing.

**Pull request merged**